File: 2015/AI/AIClient_JAVA/src/event/StartGameEvent.java
/* ------------------------------------------------------------------------------
 ** _________   _________      ________    _____      _____  ___________ _________
 ** \_   ___ \ /   _____/     /  _____/   /  _  \    /     \ \_   _____//   _____/
 ** /    \  \/ \_____  \     /   \  ___  /  /_\  \  /  \ /  \ |    __)_ \_____  \ 
 ** \     \____/        \    \    \_\  \/    |    \/    Y    \|        \/        \
 **  \______  /_______  /     \______  /\____|__  /\____|__  /_______  /_______  /
 **         \/        \/             \/         \/         \/        \/        \/ 
 **
 ** GameStartEvent.java
 ** Event that happen when a game start
 **
 ** Author: Samuel-Ricardo Carriere
 ** ------------------------------------------------------------------------------*/

package event;

import world.World;

public class StartGameEvent extends IngoingEvent {

	@Override
	public void execute() {
		World.getInstance().startGame();
	}

	@Override
	public boolean fillArguments(String message) {
		return true;
	}
}

File: 2015/AI/AIClient_Python/src/event/StartGameEvent.py
'''
Created on Dec 20, 2014

@author: scarriere
'''
from event.IngoingEvent import IngoingEvent
from aiclient.Singleton import Singleton
from world.World import World

class StartGameEvent(IngoingEvent):

    def __init__(self):
        pass

    def fillArguments(self, string):
        pass
            
    def execute(self):
        world = Singleton(World)
        world._startGame()